src/app/components/feed/FeedCollaboration.js

```diff
@@ -0,0 +1,164 @@
+  const [textValue, setTextValue] = React.useState(null);
+  const [invites, setInvites] = React.useState([]);
+
+  const handleAdd = (email) => {
+    if (EmailValidator.validate(email)) {
+      const newInvites = [...invites];
+      newInvites.push(email);
+      setTextValue(null);
+      setInvites(newInvites);
+      onChange(newInvites);
+    }
+  };
+
+  const handleDelete = (email) => {
+    const newInvites = [...invites];
+    newInvites.splice(newInvites.indexOf(email), 1);
+    setInvites(newInvites);
+  };
+
+  const FeedCollabRow = ({
+    label,
+    team,
+    type,
+  }) => (
+    <div className={styles['feed-collab-row']} key={label}>
+      { !team &&
+        <div
+          className={cx(
+            styles['feed-collab-icon'],
+            {
+              [styles.invitationSentIcon]: type === 'invitation-sent',
+            })
+          }
+        >
+          { type === 'invitation-new' && <PersonAddIcon /> }
+          { type === 'invitation-sent' && <ScheduleSendIcon /> }
+        </div>
+      }
+      { team &&
+        <TeamAvatar team={team} />
+      }
+      <div
+        className={cx(
+          styles['feed-collab-label'],
+          {
+            [styles.invitationSentLabel]: type === 'invitation-sent',
+          })
+        }
+      >
+        { type === 'invitation-sent' &&
+          <span className="typography-body1">
+            <FormattedMessage id="feedCollaboration.pending" defaultMessage="Invitation sent - pending" description="The status of a sent invitation to join a shared feed - pending acceptance" />
+            <br />
+          </span>
+        }
+        <span className={type === 'invitation-sent' ? 'typography-body2' : 'typography-body1'}>
+          {label}
+        </span>
+      </div>
+      { type === 'organizer' ?
+        <span className={cx('typography-body2', styles['feed-collab-organizer'])}>
+          <FormattedMessage id="feedCollaboration.organizer" defaultMessage="organizer" description="Label to highlight the Shared Feed organizer" />
+        </span>
+        :
+        <Tooltip arrow title={<FormattedMessage id="feedCollaboration.remove" defaultMessage="Remove" description="Tooltip for button to remove invitation or feed collaborator" />}>
+          <span>
+            <ButtonMain
+              onClick={() => handleDelete(label)}
+              iconCenter={<ClearIcon />}
+              variant="contained"
+              size="small"
+              theme="lightText"
+            />
+          </span>
+        </Tooltip>
+      }
+    </div>
+  );
+
+  return (
+    <div className={styles['feed-collab-card']}>
+      <span className="typography-subtitle2">
+        <FormattedMessage
+          id="feedCollaboration.title"
+          defaultMessage="Collaboration"
+          description="Title of the collaboration box in which feed organizer invites other organizations to a shared feed"
+        />
+      </span>
+      { !feed.dbid &&
+        <FormattedMessage
```

Comment:
can you wrap this with a class of `typography-body2` so the font sizes to match the designs?